src/modules/commands/discord/urbandictionary.ts

```diff
@@ -61,16 +67,114 @@ export default new Command<discord.Message>({
       )
 
     list = list.sort((a: Definition, b: Definition) => {
-      if (b.thumbsUp > a.thumbsUp) return 1
+      if (b.thumbsUp - b.thumbsDown > a.thumbsUp - a.thumbsDown) return 1
       else return -1
     })
 
-    let message = `**${wordString}:** ${list[0].word}\n\n`
-    message += `**Definition:** ${list[0].definition.replace(re, '*$1*')}\n\n`
+    let defN = 0
+    const definitionEmbed = new discord.MessageEmbed()
+      .setColor('#647CC4')
+      .setTitle(`${list[defN].word} (N°${defN + 1}/${list.length})`)
+      .setURL(list[defN].permalink)
+      .setAuthor('Urban dictionary', '', 'https://www.urbandictionary.com')
+      .setThumbnail('https://i.imgur.com/A6nvY85.png')
+      .addFields(
+        {
+          name: 'Definition',
+          value: list[defN].definition.replace(re, '*'),
+        },
+        { name: 'Example', value: list[defN].example.replace(re, '*') },
+      )
+      .addField('Written on', list[defN].writtenOn.substring(0, 11), true)
```

Comment:
The "Written on" field looks pretty ugly if you just take the raw timestamp. You can pass that timestamp to `new Date(timestamp)` and just extract the date (we don't need time).

Reply:
The substring will extract the date like "2021-08-11" isn't that good?

Reply:
I see, but you don't extract the substring when updating the embed below.

Reply:
And it needs to be (0, 10), what you're extracting now is "2021-08-11T".

Reply:
Still (0, 11).